### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
Commit with new design implemented

The doc inputs are pending with the new screenshots

### CODE REVIEW

File: widget/assets/css/reportAnIssue.css
Comment (cs-suyog-jinturkar) [inline widget/assets/css/reportAnIssue.css:1]: css file name need to be renamed

File: widget/assets/css/reportAnIssue.css [hunk 1]
```diff
@@ -8,24 +8,43 @@
...
   border-radius: 50%;
   color: white;
+  margin-bottom: 0px;
+  position: relative;
+  margin-right: 16px;
+}
+
+.submit-content-form-header span{
+  margin-right: 30px;
+    /* display: inline-block; */
```
Comment (cs-suyog-jinturkar) [inline widget/assets/css/reportAnIssue.css]: can we remove commented code?

File: widget/info.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
             "https://repo.fortisoar.fortinet.com/fsr-widgets/fortiAIConfiguration-1.0.0/images/uploadCustom.png",
```
Comment (cs-suyog-jinturkar) [inline widget/info.json]: this needs to be corrected

File: widget/assets/services/submitContentForm.js [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+                    }
+                ],
+                '__selectFields': [
+                    'id',
+                    'name'
+                ]
+            };
+
+            // add a tag to the playbook, get the playbook instead of hardcoding the playbook iri
+            getAllPlaybooks(queryObjectPlaybook).then(function (playbookUUID) {
```
Comment (cs-suyog-jinturkar) [inline widget/assets/services/submitContentForm.js]: This logic is used many places like CICD, outbreak, etc. Can we make it to service/controller or some directive?

File: widget/info.json [hunk 4]
```diff
@@ -18,7 +18,7 @@
...
             "https://repo.fortisoar.fortinet.com/fsr-widgets/fortiAIConfiguration-1.0.0/images/fromFsr.png"
```
Comment (cs-pranali-chavan) [inline widget/info.json]: folder name to be updated in path

File: widget/assets/css/reportAnIssue.css
Comment (cs-pranali-chavan) [reply widget/assets/css/reportAnIssue.css:1]: copyright to be added in css

**Review by cs-pranali-chavan — CHANGES_REQUESTED** [review]
comments added

File: widget/assets/services/submitContentForm.js (hunk 3 above)
Comment (cs-yash-patodkar) [reply widget/assets/services/submitContentForm.js]: Yes we can add a function in playbook service in Exo-Ui

File: widget/assets/css/reportAnIssue.css (hunk 1 above)
Comment (cs-yash-patodkar) [reply widget/assets/css/reportAnIssue.css]: done

File: widget/info.json (hunk 2 above)
Comment (cs-yash-patodkar) [reply widget/info.json]: done

File: widget/info.json (hunk 4 above)
Comment (cs-yash-patodkar) [reply widget/info.json]: done

File: widget/assets/css/reportAnIssue.css
Comment (cs-yash-patodkar) [reply widget/assets/css/reportAnIssue.css:1]: done

File: widget/assets/services/submitContentForm.js [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+                        "field": "installed",
+                        "operator": "eq",
+                        "value": true
+                    }
+                ]
+            }
+        }
+
+        function exportSolution(contentDetail, scope) {
+            scrollToBottom();
```
Comment (cs-suyog-jinturkar) [inline widget/assets/services/submitContentForm.js]: Export functionality can be a separate service from re-usability point of view

File: widget/assets/services/submitContentForm.js [hunk 6]
```diff
@@ -0,0 +1,328 @@
...
+                        scope.uploadedFileFlag = true;
+                        triggerPlaybook(scope);
+                    },
+                        function (response) {
+                            scope.loadingJob = false;
+
+                            if (response.status > 0) {
+                                $log.debug(response.status + ': ' + response.data);
+                            }
+                            var message = 'Upload failed. Please try again.';
```
Comment (cs-suyog-jinturkar) [inline widget/assets/services/submitContentForm.js]: i18n impact?

File: widget/assets/services/submitContentForm.js [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+                            if (response.status === 413) {
+                                message = 'File exceeds the maximum size.';
+                            }
+                            toaster.error({ body: message });
+                        });
+                }
+            }
+            else {
+                toaster.error({ body: 'File size exceeded limit, please try again' });
```
Comment (cs-suyog-jinturkar) [inline widget/assets/services/submitContentForm.js]: move all errors in a msg const

File: widget/assets/services/submitContentForm.js (hunk 3 above)
Comment (cs-suyog-jinturkar) [reply widget/assets/services/submitContentForm.js]: @cs-pranali-chavan to track

File: widget/view.controller.js [hunk 8]
```diff
@@ -104,15 +161,24 @@ Copyright end */
...
       }
     }
 
-    function closeNote() {
-      var disclaimerBox = document.getElementById('disclaimer-box');
-      disclaimerBox.remove();
-      var formDetails = document.getElementById('community-details-form');
-      formDetails.setAttribute('style', 'margin-top: 0px; height: 865px;');
+    function openDocumentation(){
+      $window.open('https://github.com/fortinet-fortisoar/widget-submit-content-form/blob/release/1.0.0/docs/usage.md#prompting-tips', '_blank');
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js:168]: is this link correct?

File: widget/view.controller.js [hunk 9]
```diff
@@ -46,26 +47,82 @@ Copyright end */
         'outdent', 'divider', 'table', 'image', 'link', 'divider', 'code', 'codeblock'
       ]
     };
-    $scope.nextPage = false;
+    $scope.moveToStep = moveToStep;
+    $scope.openDocumentation = openDocumentation;
+
+    $scope.showCreatedSolutions = 'created';
+    $scope.currentStep = 1;
+    // $scope.letsGetStarted = letsGetStarted;
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js]: commented code

File: widget/view.controller.js [hunk 10]
```diff
@@ -104,15 +161,24 @@ Copyright end */
...
+
+    function minimize() {
+      const customModal = document.getElementById('custom-modal');
+      customModal.setAttribute('style', 'display:none;');
+      $scope.submitContentFormForm.$setPristine();
+      $scope.submitContentFormForm.$setUntouched();
+    }
+
+    function moveToStep(stepNumber) {
+      //$scope.currentStep = 3
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js:179]: commented code

File: .vscode/settings.json [hunk 11]
```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
```
Comment (cs-suyog-jinturkar) [inline .vscode/settings.json]: Do we need this file?

File: widget/view.controller.js (hunk 8 above)
Comment (cs-yash-patodkar) [reply widget/view.controller.js:168]: yes

File: widget/view.controller.js (hunk 10 above)
Comment (cs-yash-patodkar) [reply widget/view.controller.js:179]: For testing

**Review by cs-pranali-chavan — APPROVED** [review]
Looks good

**Pull request merged**